static/monsterSort.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	$('select').change(function() {
+		selection = $(this).val();
+		if (selection == 'all') {
+			$('.monster-box').show();
+		} else {
+			var selector = '.' + selection;
```

Comment:
That's one option -- or you could use a compound selector like `.monster-box.good`.